README.md

```diff
@@ -128,284 +43,8 @@
   # Amount to take is greater than the number of elements
   ```
 
-* <a name="space-before-anonymous-fun-arrow"></a>
-  Always use a space before `->` in 0-arity anonymous functions.
-  <sup>[[link](#space-before-anonymous-fun-arrow)]</sup>
-
-  ```elixir
-  # Bad
-  Task.async(fn->
-    ExUnit.Diff.script(left, right)
-  end)
-
-  # Good
-  Task.async(fn ->
-    ExUnit.Diff.script(left, right)
-  end)
-  ```
-
-### Indentation
-
-* <a name="binary-ops-indentation"></a>
```

Comment:
The formatter also handles indentation but with slightly different rules. For example, `when` works like other binary operators such as `++` (and not like `|>`). The pipe rule has also been generalized to any other pipe like operator.